runtime/commonMain/src/kotlinx/serialization/SerialFormat.kt

```diff
@@ -12,7 +12,7 @@ interface SerialFormat {
 }
 
 @Deprecated(
-    "AbstractSerialFormat is deprecated for removal since it is indistinguishable from SerialFormat interface." +
+    "Deprecated for removal since it is indistinguishable from SerialFormat interface. " +
             "Use SerialFormat instead.", ReplaceWith("SerialFormat"), DeprecationLevel.WARNING
```

Comment:
Let's make it `DeprecationLevel.ERROR` already?

Reply:
There was no release with WARNING yet. Moreover, this class is not really harmful and hardly can be accidentaly misused (as opposed to internal functions), and used in 3rd party format implementations.